Restore.cmd

```diff
@@ -1,3 +1,3 @@
-@if not defined EchoOn @echo off
-powershell -noprofile -executionPolicy RemoteSigned -file "%~dp0\build\scripts\restore-legacy.ps1" %*
+@echo off
```

Comment:
All instances of `@echo off` should be replaced with `@if not defined _echo @echo off`. This assists in debugging without having to go touch a bunch of files.

CoreCLR, CoreFX, and Projecy System are currently using `_echo`. I believe we have some internal scripts which are using `EchoOn`